manifests/params.pp

```diff
@@ -322,6 +322,8 @@
   $agent_tlsaccept                          = undef
   $agent_tlscafile                          = undef
   $agent_tlscertfile                        = undef
+  $agent_tlscertissuer                      = undef
+  $agent_tlscertsubject                     = undef
```

Comment:
I think these can be removed and add the default value in `agent.pp` since it's the only place that they are being used.

Reply:
I’m not sure what is the etiquette about resolving the conversations, should I resolve the conversation if the code is changed as the comment suggested, or should I let you close the conversation? :innocent:

Reply:
You can always resolve it and i'll got through it anyway 😄